options/ansi/generic/time-stubs.cpp

```diff
@@ -334,9 +334,25 @@ int clock_settime(clockid_t, const struct timespec *) {
 	__builtin_unreachable();
 }
 
-int utimes(const char *, const struct timeval[2]) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+int utimes(const char *filename, const struct timeval[2]) {
+	if (!mlibc::sys_utimensat) {
+		MLIBC_MISSING_SYSDEP();
+		errno = ENOSYS;
+		return -1;
+	}
+	// TODO: We really should respect timeval here
```

Comment:
We can just check if ``times != NULL`` here and convert it to a timespec right? Seems simple enough and it's a good idea to include it else we might produce unexpected results silently